client/verta/docs/examples/tutorials.rst

```diff
@@ -1,12 +1,15 @@
 Tutorials
 =========
 
-Check out these tutorials for detailed step-by-step instructions on performing different MLOps 
+Check out these tutorials for detailed step-by-step instructions on performing different MLOps
 functions in Verta.
 
 ModelDB (Versioning and Metadata)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+* `Versioning model ingredients <tutorials/ingredients.html>`_, which allows you to have Git-like
```

Comment:
The next URL was a placeholder for this one -- you can just. reuse that

Reply:
I would recommend keeping the versioning of ingredients (via git-like) separate from versioning the model (via run and logging things) separate right now as they are pretty different. It also reinforces the idea that it's something that can be done separate from the training itself.